src/playercards/cards/WellConnected.ttslua

```diff
@@ -0,0 +1,52 @@
+local playmatApi = require("playermat/PlaymatApi")
+
+local display = false
+local count = 0
+local modValue = 5
+local loopId = nil
+
+local b_display = {
+    index = 0,
+    click_function = 'none',
+    function_owner = self,
+    position = {-0.25,1,-0.75},
+    font_color = {1,1,1,100},
+    font_size = 250,
+    width = 0,
+    height = 0
+}
+
+function onLoad()
+  self.addContextMenuItem('Toggle Counter', toggleCounter)
+end
```

Comment:
This needs an `onSave()` function to remeber the context menu selection after saving & loading.